cartridge/lua-api/compression.lua

```diff
@@ -0,0 +1,171 @@
+local lua_api_get_topology = require('cartridge.lua-api.get-topology')
+
+local log = require('log')
+local pool = require('cartridge.pool')
+local errors = require('errors')
+
+-- Surf by replicates to find master storages and call on them getStorageCompressionInfo()
+local function get_cluster_compression_info()
+    local replicasets, err = lua_api_get_topology.get_replicasets()
+    if replicasets == nil then
+        log.error(err)
+        return nil, err
+    end
+
+    local compression_info = {}
+
+    for _, rpl in pairs(replicasets or {}) do
+        for _, role in pairs(rpl.roles or {}) do
+            if role == 'vshard-storage' then
+                local master = rpl.master
+
+                local storage_compression_info, err = errors.netbox_call(
+                    pool.connect(master.uri, {wait_connected = true}),
+                    '_G.getStorageCompressionInfo', {master}, {timeout = 1}
+                )
+                if storage_compression_info == nil then
+                    error(err)
+                end
+
+                table.insert(compression_info, {
+                    instance_id = master.uuid,
+                    instance_compression_info = storage_compression_info[1]})
+            end
+        end
+    end
+
+    return {
+        compression_info = compression_info,
+    }
+end
+
+local function create_test_space(space_name, orig_space, field_format)
+    if box.space[space_name] ~= nil then
+        box.space[space_name]:drop()
+    end
```

Comment:
Please change function signature:
```suggestion
local function create_test_space(space_name, space_type, field_format)
        if space_type == '' then
                error(...)
        end
        local tmp_space = space_name..space_type
        if box.space[tmp_space] ~= nil then
        box.space[tmp_space]:drop()
    end
```
It makes the code more secure from dropping production spaces with data